### PR TITLE
Francisca Fragmentation Ammo

### PR DESCRIPTION
**Pull request by TheDanseMacabre**
Adds a type of Francisca ammo that has lower penetration and individual damage compared to FMJ and AP Francisca ammo, but spawns the equivalent of a frag grenade's explosion on impact.

### CODE REVIEW

**Comment by TheDanseMacabre** [comment]
!review

File: code/modules/overmap/ship_weaponry/weaponry/francisca.dm [hunk 1]
```diff
@@ -28,20 +28,39 @@
...
 	name_override = "40mm AP burst"
-	desc = "A box of AP bullets for use in a Francisca rotary gun."
+	desc = "A box of AP shells for use in a Francisca rotary gun."
 	icon_state = "box_ap"
 	impact_type = SHIP_AMMO_IMPACT_AP
 	projectile_type_override = /obj/item/projectile/ship_ammo/francisca/ap
 
+/obj/item/ship_ammunition/francisca/frag
+	name = "40mm fragmentation ammunition box"
+	name_override = "40mm FRAG burst"
```
Comment (SleepyGemmy) [inline code/modules/overmap/ship_weaponry/weaponry/francisca.dm:38]:
```suggestion
	name_override = "40mm frag burst"
```
or fragmentation.

File: code/modules/overmap/ship_weaponry/weaponry/francisca.dm [hunk 2]
```diff
@@ -28,20 +28,39 @@
...
 
 /obj/item/projectile/ship_ammo/francisca/ap
-	name = "40mm AP bullet"
+	name = "40mm AP shell"
 	damage = 30
 	armor_penetration = 100
 	penetrating = 4
+
+/obj/item/projectile/ship_ammo/francisca/frag
+	name = "40mm FRAG shell"
```
Comment (SleepyGemmy) [inline code/modules/overmap/ship_weaponry/weaponry/francisca.dm:58]:
```suggestion
	name = "40mm frag shell"
```
or fragmentation.

File: code/modules/overmap/ship_weaponry/weaponry/francisca.dm [hunk 3]
```diff
@@ -28,20 +28,39 @@
...
+	damage = 30
+	armor_penetration = 50
+	penetrating = 1
+
+
```
Comment (SleepyGemmy) [inline code/modules/overmap/ship_weaponry/weaponry/francisca.dm:63]:
```suggestion
```

File: code/modules/overmap/ship_weaponry/weaponry/francisca.dm [hunk 4]
```diff
@@ -28,20 +28,39 @@
...
+/obj/item/projectile/ship_ammo/francisca/frag/on_impact(var/atom/A)
+	fragem(src, 70, 70, 1, 2, 10, 4,TRUE)
```
Comment (SleepyGemmy) [inline code/modules/overmap/ship_weaponry/weaponry/francisca.dm:65]:
```suggestion
	fragem(src, 70, 70, 1, 2, 10, 4, TRUE)
```

**Review by SleepyGemmy — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**